public_suffix_list.dat

```diff
@@ -12635,7 +12635,9 @@ on-web.fr
 
 // Platform.sh : https://platform.sh
 // Submitted by Nikola Kotur <nikola@platform.sh>
-*.platform.sh
+*.bc.platform.sh
+*.eu.platform.sh
+*.us.platform.sh
```

Comment:
```suggestion
bc.platform.sh
ent.platform.sh
eu.platform.sh
us.platform.sh
```
Adding `ent.` and removing the wildcard: customer sites are directly below these 4 suffixes.

Reply:
You are right, @pjcdawkins, updating this PR now.